### PR TITLE
Feature/pop up card css

### PR DESCRIPTION
**Pull request by michellekaplan7**
## Type of change made:
 - [ ] Bugfix (non-breaking change which fixes an issue)
 - [ ] New feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
 - [X] Styling- no new features

## Detailed Description

Added styling to the pop-up-card upon clicking the update buttons on the top in the nav. Also added some styling to the buttons in the pop-up-card along with the buttons in the nav. This is just initial styling. This may change with Sass.

Files modified:
 - [X] index.html
 - [X] styles.scss
 - [ ] base.scss
 - [X] scripts.js
 - [ ] User.js
 - [ ] User-repo.js
 - [ ] Activity.js
 - [ ] Hydration.js
 - [ ] Sleep.js
 
 
 Other files:

### CODE REVIEW

File: src/css/style.scss [hunk 1]
```diff
@@ -279,3 +285,32 @@ h1 {
...
 }
+
+//form and pop-up-card
+.pop-up-card {
+  background-color: #9a9dab;
+  border: 2px solid black;
+  height: 20em;
+  margin-left: 3em;
+  position: absolute;
+  right: 200px;
```
Comment (ttarlov) [inline src/css/style.scss:296]: We should later figure out how to make the pop-up card stay a bit more in the middle as its a little skewed to the right. 
image

File: src/css/style.scss [hunk 2]
```diff
@@ -3,39 +3,45 @@
...
+  background-color: steelblue;
+  border-radius: 1em;
+  color: white;
+  cursor: pointer;
+  font-weight: bold;
+  height: 3em;
+  width: 14em;
+}
+
+button:hover {
```
Comment (ttarlov) [inline src/css/style.scss:34]: nice hover effects! looks good.

**Comment by ttarlov** [comment]
all looks great! I noticed that when the pop up is up on the screen and you click any of the buttons in the nav nothing happens. I think one way to get around this is to just blur the nav area too or we can fix the button functionality.

File: src/css/style.scss (hunk 1 above)
Comment (carflor) [reply src/css/style.scss:296]: Agreed. We tried a couple different things to flex it ... but the position: absolute (our best guess) kept leaving it unchanged. So we had to hard code it which means that in any different sized brower it looks awkward. Happy to try to figure a way around it... i wonder if grid would work, we did not try that approach.

File: src/css/style.scss (hunk 2 above)
Comment (carflor) [reply src/css/style.scss:34]: Love the button changes. Great job Michelle.

**Comment by carflor** [comment]
good eye for detail Taras. A blur on the nav sounds good as a way to force user attention on form.

**Pull request merged**